bindinfo/bind_test.go

```diff
@@ -2114,3 +2115,38 @@ func (s *testSuite) TestTemporaryTable(c *C) {
 	tk.MustGetErrCode("create binding for update t set a = 1 where b = 1 and c > 1 using update /*+ use_index(t, c) */ t set a = 1 where b = 1 and c > 1", errno.ErrOptOnTemporaryTable)
 	tk.MustGetErrCode("create binding for delete from t where b = 1 and c > 1 using delete /*+ use_index(t, c) */ from t where b = 1 and c > 1", errno.ErrOptOnTemporaryTable)
 }
+
+func (s *testSuite) TestBindingLastUpdateTime(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	s.cleanBindingEnv(tk)
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists t0;")
+	tk.MustExec("create table t0(a int, key(a));")
+	tk.MustExec("create global binding for select * from t0 using select * from t0 use index(a);")
+	tk.MustExec("admin reload bindings;")
+
+	rows := tk.MustQuery("show global bindings").Rows()
+	c.Assert(len(rows), Equals, 1)
+	updateTime := rows[0][5]
+	updateTimeVal, ok := updateTime.(string)
+	c.Assert(ok, Equals, true)
+	// The format of update time in global binding is 'xxxx-xx-xx xx:xx:xx.xxx'.
+	updateTimeStr := strings.Split(updateTimeVal, ".")
+
+	// The format of update time in stats var is 'xxxx-xx-xx xx:xx:xx'. And it will convert to 'xxxx-xx-xx xx:xx:xx.000'.
+	// So when we compare, we remove the last three digits and compare.
+	rows1 := tk.MustQuery("show status like 'last_plan_binding_update_time';").Rows()
+	updateTime1 := rows1[0][1]
+	updateTimeVal1, ok := updateTime1.(string)
+	c.Assert(ok, Equals, true)
+	updateTimeStr1 := strings.Split(updateTimeVal1, ".")
+	c.Assert(updateTimeStr1[0], Equals, updateTimeStr[0])
```

Comment:
This test is unstable. I'll fix it ASAP.

Reply:
Why will it be unstable?

Reply:
Update, the unstable test is fixed. Previous I use the `show global binding` to get the last update time. But the time format is not the same here(I have no idea why this happens in the test. If we use the MySQL Client to execute the same query we will get the same time format).